pages/team.html

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+permalink: /team/
+title: "The Privacy&#8203;Tools Team"
+description: "Core contributors to the Privacy&#8203;Tools website, communities, and services."
+---
+
+<div class="row">
+
+{% include team.html
+avatar="burung.jpg"
+name="Burung Hantu"
+role="Founder"
+bio="I founded PrivacyTools in 2015 to share resources and tools, allowing users to avoid international mass-surveillance programs."
+email="dude@privacytools.io"
+mastodon="https://social.privacytools.io/@BurungHantu"
+twitter="privacytoolsIO"
+%}
+
+{% include team.html
+avatar="jonah.png"
+name="Jonah Aragon"
+nick="Jonah"
+role="Administrator"
+bio="I run the website and services for PrivacyTools. My goal is to spread the word about data privacy as widely as possible."
+email="jonah@privacytools.io"
+website="https://www.jonaharagon.com"
+mastodon="https://social.privacytools.io/@jonah"
+twitter="JonahAragon"
+blog="jonah"
+%}
+
+{% include team.html
+avatar="blacklight447.png"
+name="blacklight447"
+bio="I research new privacy recommendations and moderate our communities. My expertise is endpoint security and networking."
+role="Community Manager"
+email="blacklight447@privacytools.io"
+mastodon="https://social.privacytools.io/@blacklight447"
+%}
+
+{% include team.html
+avatar="danarel.jpg"
+name="Dan Arel"
+role="ThinkPrivacy Editor"
+bio="I'm an author, columnist, and privacy advocate whose work has appeared in Time, Huff Post, OpenSource, and more. I am editing ThinkPrivacy, a PrivacyTools resource for everyday users."
+website="https://www.danarel.com"
+twitter="danarel"
+mastodon="https://mastodon.technology/@danarel"
```

Comment:
```suggestion
mastodon="https://mastodon.technology/@danarel"
blog="danarel"
```

Forgot I needed to add this (links to https://blog.privacytools.io/author/danarel/)